### PR TITLE
[Chains] Update chains: Linea Mainnet

### PR DESCRIPTION
**Pull request by kumaryash90**

### CODE REVIEW

File: packages/chains/data/additional/59144.mjs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  chain: "Linea Mainnet",
+  shortName: "linea-mainnet",
+  chainId: 59144,
+  testnet: false,
+  nativeCurrency: {
+    name: "Ether",
+    symbol: "ETH",
+    decimals: 18,
+  },
+  rpc: ["https://rpc.linea.build"],
```
Comment (jnsdls) [inline packages/chains/data/additional/59144.mjs:16]: don't add this yet since it doesn't work (otherwise RPC edge will attempt to use this)

**Review by jnsdls — COMMENTED** [review]
@kumaryash90 one small change then shipit

File: packages/chains/data/additional/59144.mjs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  },
+  rpc: [],
+  explorers: [
+    {
+      name: "Linea Scan",
+      url: "https://lineascan.build",
+      standard: "",
+    },
+  ],
+  faucets: ["https://www.infura.io/faucet/linea"],
```
Comment (nachoiacovino) [inline packages/chains/data/additional/59144.mjs:24]: faucet for mainnet?

Comment (kumaryash90) [reply packages/chains/data/additional/59144.mjs:24]: lol, they filled it in the form

**Pull request merged**